Doc/library/shlex.rst

```diff
@@ -61,6 +61,18 @@ The :mod:`shlex` module defines the following functions:
    string that can safely be used as one token in a shell command line, for
    cases where you cannot use a list.
 
+   .. warning::
+
+      The ``shlex`` module is **only designed for Unix shells**.
+
+      The :func:`quote` function is not garaunteed to be safe on non-compliant
```

Comment:
Hmm, "non-compliant" to what?

Reply:
Whoops, this was meant to be POSIX compliance since that's the basic specification that the quoting rules in shlex follow (and what the comments in the file say).